packages/hooks/__tests__/use-lockscreen.test.tsx

```diff
@@ -20,6 +20,12 @@ const Comp = defineComponent({
   },
 })
 
+function waitKlsUpdate(ms = 250) {
+  return new Promise<void>((resolve) => {
+    setTimeout(resolve, ms)
+  })
+}
+
```

Comment:
There is an implemented `sleep` function inside the `test-utils` folder, which can be directly imported and used.
https://github.com/element-plus/element-plus/blob/22ac4c56f2360ba2bb98153c274a18fedac511d8/packages/test-utils/sleep.ts#L1-L3